admin-auth/src/test/kotlin/com/epam/drill/admin/auth/UserRepositoryImplTest.kt

```diff
@@ -15,19 +15,234 @@
  */
 package com.epam.drill.admin.auth
 
-import org.mockito.MockitoAnnotations
+import com.epam.drill.admin.auth.entity.UserEntity
+import com.epam.drill.admin.auth.principal.Role
+import com.epam.drill.admin.auth.repository.impl.UserRepositoryImpl
+import com.epam.drill.admin.auth.table.UserTable
+import kotlinx.coroutines.runBlocking
+import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.statements.InsertStatement
+import org.jetbrains.exposed.sql.transactions.experimental.newSuspendedTransaction
+import org.jetbrains.exposed.sql.transactions.transaction
 import kotlin.test.*
 
 class UserRepositoryImplTest {
+
+    private val repository = UserRepositoryImpl()
```

Comment:
I suggest renaming `UserRepositoryImpl` to something like `UserRepositorySqlDb` (name up to discussion) to reflect backing mechanism in the title